src/create-reminder-signup/lib/fetch-with-retry.ts

```diff
@@ -0,0 +1,12 @@
+import { Response } from 'node-fetch';
+
+export async function fetchWithRetry(
+	fetcher: () => Promise<Response>,
+	maxTries: number,
+): Promise<Response> {
+	const response = await fetcher();
```

Comment:
I'm being pedantic here, but calling it with `maxTries` of 0 will still make a request! Would it make sense to rename it as `maxRetries`, and the condition becomes `maxRetries > 0`?

Reply:
No I love that level of pedantry! Yeah I agree 👍 I'll make that change now.